examples/experimental/PointersAhead.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
Maybe remove this notebook from the PR as it is unrelated

Reply:
deleted the notebook